test/integration/tls/ciphers_test.go

```diff
@@ -61,6 +61,9 @@ func runBasicSecureAPIServer(t *testing.T, ciphers []string) (uint32, error) {
 		kubeAPIServerOptions.InsecureServing.BindPort = 0
 		kubeAPIServerOptions.Etcd.StorageConfig.ServerList = []string{framework.GetEtcdURL()}
 		kubeAPIServerOptions.ServiceClusterIPRange = *defaultServiceClusterIPRange
+		if err := app.DefaultOptions(kubeAPIServerOptions); err != nil {
```

Comment:
we really should switch this to the testserver.

Reply:
testserver is in k8s.io/apiextensions-apiserver/test/integration/testserver not for kube-apiserver

Reply:
Oh,  you mean this `cmd/kube-apiserver/app/testing/testserver.go`

will have a try with it.

Reply:
done